db/active_replicator_push.go

```diff
@@ -180,16 +186,9 @@ func (apr *ActivePushReplicator) _initCheckpointer(remoteCheckpoints []replicati
 	return nil
 }
 
-// GetStatus is used externally to retrieve pull replication status.  Combines current running stats with
-// initialStatus.
-func (apr *ActivePushReplicator) GetStatus() *ReplicationStatus {
-	return apr.getPushStatus(apr.getCheckpointHighSeq())
-}
-
-// getPullStatus is used internally, and passed as statusCallback to checkpointer
-func (apr *ActivePushReplicator) getPushStatus(lastSeqPushed string) *ReplicationStatus {
+func (apr *ActivePushReplicator) _getStatus() *ReplicationStatus {
```

Comment:
Can we note here that `_getStatus()` needs to be called apr.stateLock already acquired?